README.md

```diff
@@ -34,36 +35,68 @@ npm install --save @webex/components
 
 ## Usage
 
-We need to externally import all the fonts, icons, images and core _CSS_ manually. Import `@webex/components/dist/webexComponents.css` into your main entry file. There are two ways to do this:
+### Styles
 
-### 1. JavaScript
+In order to properly style Webex Components, we need to import all the fonts, icons, images and core _CSS_ manually.
+Import `@webex/components/dist/webexComponents.css` into your main entry file.
+
+There are two ways to do this:
+
+#### JavaScript
+
+In your `index.js`, add the following import:
 
 ```js
 import '@webex/components/dist/webexComponents.css';
+...
 ```
 
-### 2. Html
+#### HTML
+
+In the `<head>` of your `index.html`, add the following import:
 
 ```html
 <head>
+  ...
   <link rel="stylesheet" type="text/css" href="node_modules/@webex/components/dist/webexComponents.css" />
 </head>
 ```
 
-Once Webex styles have been imported into your project, you can start using the components. For example:
+### Adapters
+Webex Components are self-updating, meaning, they know how to fetch the data they need.
+Data is passed to components via adapter classes (see [adapters](./src/adapters)).
+Adapters are an uniform interface for the Webex Components to consume.
+They also know how to map the data from their data source to the data the components need.
 
+To use a Webex Component, start by creating a Webex Adapter:
 ```js
-import {WebexAvatar} from '@webex/components';
+import '@webex/components/dist/webexComponents.css';
```

Comment:
I dont think this is necessary. you have it imported in your components file.

Reply:
What components file?
The idea is that the sections build on each other: style, style + adapter, style + adapter + component

Reply:
this looks like you are making this file an entry file, which does not have to be. it looks a bit confusing to me. You are introducing the adapter but adding extra code about style import declaration which is not relevant to your description.

Reply:
we are addressing the styling import declaration in the earlier step with two ways. and we should leave it at that.

Reply:
this means that you are basically assuming that all these declaration will occur in `index.js` which potentially will not be the case.